pkg/template/api/validation/validation.go

```diff
@@ -25,37 +26,16 @@ func ValidateParameter(param *api.Parameter) (errs errors.ValidationErrorList) {
 
 // ValidateTemplate tests if required fields in the Template are set.
 func ValidateTemplate(template *api.Template) (errs errors.ValidationErrorList) {
-	if len(template.Name) == 0 {
+	if len(template.ObjectMeta.Name) == 0 {
```

Comment:
ObjectMeta shouldn't be needed here... the way it was before should still work since ObjectMeta is embedded